File: token-validation-jaxrs/src/test/java/no/nav/security/token/support/jaxrs/rest/UnprotectedClassResource.java
package no.nav.security.token.support.jaxrs.rest;

import jakarta.ws.rs.GET;
import jakarta.ws.rs.Path;
import jakarta.ws.rs.core.Response;
import no.nav.security.token.support.core.api.Unprotected;

@Path("class/unprotected")
@Unprotected
public class UnprotectedClassResource {

    @GET
    public Response get() {
        return Response.ok().build();
    }

}

File: token-validation-jaxrs/src/test/kotlin/no/nav/security/token/support/jaxrs/rest/UnprotectedClassResource.kt
package no.nav.security.token.support.jaxrs.rest

import jakarta.ws.rs.GET
import jakarta.ws.rs.Path
import jakarta.ws.rs.core.Response
import jakarta.ws.rs.core.Response.*
import no.nav.security.token.support.core.api.Unprotected

@Path("class/unprotected")
@Unprotected
class UnprotectedClassResource {
    @GET
    fun get() = ok().build()
}